cvat-ui/src/reducers/models-reducer.ts

```diff
@@ -15,27 +17,52 @@ const defaultState: ModelsState = {
     detectors: [],
     trackers: [],
     reid: [],
+    classifiers: [],
     modelRunnerIsVisible: false,
     modelRunnerTask: null,
     inferences: {},
+    totalCount: 0,
+    query: {
+        page: 1,
+        id: null,
+        search: null,
+        filter: null,
+        sort: null,
+    },
+    providers: {
+        fetching: false,
+        current: [],
```

Comment:
let's rename to just ``list`` or something like this, because ``current`` is better to be used with paginatable resources